kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-core/src/main/java/org/kie/server/services/taskassigning/core/model/ModelConstants.java

```diff
@@ -57,7 +57,7 @@ private ModelConstants() {
                                                             Collections.unmodifiableMap(new HashMap<>()),
                                                             false,
                                                             Collections.unmodifiableSet(new HashSet<>(Collections.singletonList(PLANNING_USER))),
-                                                            Collections.unmodifiableSet(new HashSet<>()));
+                                                            Collections.unmodifiableMap(new HashMap<>()));
```

Comment:
`Collections.emptyMap()` will do the same thing, and will save a lot of memory in the process.